src/backend/mailbox_manager/api/client/serializers.py

```diff
@@ -31,7 +31,12 @@ def create(self, validated_data):
         """
         Override create function to fire a request on mailbox creation.
         """
-        mailbox_status = enums.MailDomainStatusChoices.PENDING
+        if models.Mailbox.objects.filter(
+            local_part=validated_data["local_part"]
+        ).exists():
+            raise exceptions.ValidationError(
+                {"__all__": ["Mailbox with this Local_part and Domain already exists."]}
```

Comment:
This exception message mentions "local_part and domain" but the code actually only filters on local_part.

So if I have management access to more than one domain, it seems like trying to create a mailbox with the same local part as a mailbox that exists on *any* domain will raise this error ? The test doesn't seem to make this distinction.

I'm not very familiar with serializers so I might be missing something here.